package/obs/rmt-server.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Mar 12 11:20:00 UTC 2024 - Zuzana Petrova <zpetrova@suse.com>
+
+- make sure yum that can read repomd.xml correctly is installed (bsc#1221223)
```

Comment:
Please move this down to the `2.15` release section

Reply:
Natnael, thank you, but it still says "Merging is blocked, review required"